f/radio/fn_radioCheckChannels.sqf

```diff
@@ -28,7 +28,7 @@ for "_i" from 1 to 10 do {
 	
 	// Check for vehicles. Don't add send permissions unless they're the driver.
 	_channelVehicles = (f_var_radioChannelsVehicles get _i);
-	if ((vehicle player) in _channelVehicles) then {
+	if ((str vehicle player) in _channelVehicles) then {
```

Comment:
I'm not certain `str` always returns the variable name of an object. is this documented somewhere? It's possible that `vehicleVarName` can be used.

Reply:
If a vehicle has a variable name, str returns it; if it doesn't, it returns an identifier something like `B Alpha 1-1:1 NikkoJT`, which is fine since we shouldn't be matched it if it has no variable name. I tested it.

Reply:
Oh of course it was only "documented" in the [community-provided notes](https://community.bistudio.com/wiki/str#Notes): `When applied to a unit, this returns the variable name that was assigned to the unit in the editor.`